scripts/custom_lint_checks.py

```diff
@@ -269,6 +269,14 @@ def visit_functiondef(self, node):
         self.check_functiondef_yields(node, node_doc)
 
     def check_functiondef_params(self, node, node_doc):
+        """Checks whether all parameters in a function definition are documented.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or
+                method definition in the AST.
+            node_doc: Pylint docstring class. It represents an AST node's
```

Comment:
Phrase this like:
"node_doc: Docstring. Pylint Docstring class instance representing a node's docstring."

Emulate this for all *_doc args

Reply:
done.